_locales/en/messages.json

```diff
@@ -1755,7 +1755,7 @@
 		"message": "No trackers blocked"
 	},
 	"hub_setup_blocking_text_everything": {
-		"message": "Block Everything"
+		"message": "Block Everyting"
```

Comment:
This is the wrong string.  Revert to the old copy here.
Everything is spelled wrong